quantum/process_keycode/process_unicode.c

```diff
@@ -17,15 +17,10 @@
 #include "action_util.h"
 #include "eeprom.h"
 
-static uint8_t first_flag = 0;
-
 bool process_unicode(uint16_t keycode, keyrecord_t *record) {
   if (keycode > QK_UNICODE && record->event.pressed) {
-    if (first_flag == 0) {
-      set_unicode_input_mode(eeprom_read_byte(EECONFIG_UNICODEMODE));
-      first_flag = 1;
-    }
     uint16_t unicode = keycode & 0x7FFF;
+    get_unicode_input_mode();
```

Comment:
Same goes here. This call would be replaced with `prepare_unicode_input_mode();`.